tests/visualize/analysis/stmt/parser/test_if_stmt.py

```diff
@@ -0,0 +1,134 @@
+import ast
+from unittest.mock import MagicMock, patch
+
+import pytest
+
+from app.visualize.analysis.element_manager import CodeElementManager
+from app.visualize.analysis.stmt.expr.expr_traveler import ExprTraveler
+from app.visualize.analysis.stmt.expr.model.expr_obj import CompareObj
+from app.visualize.analysis.stmt.model.if_stmt_obj import IfConditionObj, ElifConditionObj, ElseConditionObj
+from app.visualize.analysis.stmt.parser.if_stmt import IfStmt
+
+
+@pytest.fixture
+def create_ast():
+    # 코드를 ast 노드로 변환하는 함수를 반환
+    def _create_ast_node(code):
+        # ast.Call 반환
+        return ast.parse(code).body[0]
+
+    return _create_ast_node
+
+
+@pytest.mark.parametrize(
+    "input_code, expected, travel_return_value, evaluate_test_value_return",
+    [
+        pytest.param(
+            "a>10",
+            IfConditionObj(id=1, expressions=("a>10",), result=False),
+            CompareObj(expressions=("a>10",), value="10>10"),
+            False,
+            id="a>10",
+        ),
+        pytest.param(
+            "a>b",
+            IfConditionObj(id=1, expressions=("a>b", "10>9"), result=True),
+            CompareObj(expressions=("a>b", "10>9"), value="10>9"),
+            True,
+            id="a>b",
+        ),
+    ],
+)
+def test_parse_if_condition(
+    input_code, expected, travel_return_value, evaluate_test_value_return, create_ast, elem_manager
+):
+    test_node = create_ast(input_code).value
+    with (
+        patch.object(ExprTraveler, "travel", return_value=travel_return_value),
+        patch.object(IfStmt, "_evaluate_test_value", return_value=evaluate_test_value_return),
+    ):
+        assert IfStmt.parse_if_condition(test_node, elem_manager) == expected
+
+
+@pytest.mark.parametrize(
+    "input_code, expected, travel_return_value, evaluate_test_value_return",
+    [
+        pytest.param(
+            "a>10",
+            ElifConditionObj(id=1, expressions=("a>10",), result=False),
+            CompareObj(expressions=("a>10",), value="10>10"),
+            False,
+            id="a>10",
+        ),
+        pytest.param(
+            "a>b",
+            ElifConditionObj(id=1, expressions=("a>b", "10>9"), result=True),
+            CompareObj(expressions=("a>b", "10>9"), value="10>9"),
+            True,
+            id="a>b",
+        ),
+    ],
+)
+def test_parse_elif_condition(
+    input_code, expected, travel_return_value, evaluate_test_value_return, create_ast, elem_manager
+):
+    test_node = create_ast(input_code).value
```

Comment:
공통적으로 value에 밑줄이 그어져 있네요
```suggestion
    expr_node: ast.Expr = create_ast(input_code)
    test_node = expr_node.value
```
이런식으로 바꾸는건 어떨까요? 이렇게 바꾸려면 create_ast - fixture에  -> ast를 붙여주면 됩니다

Reply:
지금보니 밑줄이 없어졌습니다~